php-nginx/Dockerfile

```diff
@@ -17,12 +17,14 @@
 FROM gcr.io/google_appengine/base
 
 # persistent / runtime deps
+RUN echo "deb http://http.debian.net/debian jessie-backports main" >> /etc/apt/sources.list
```

Comment:
We can also [compile it](https://github.com/grpc/grpc/blob/master/INSTALL#L25), since who knows when that will happen...